src/routes/blog/post/announcing-messaging-set-up-a-messaging-service

```diff
@@ -0,0 +1,73 @@
+
+## The power of communication
+
+Effective communication with your users is key to high user engagement and retention. Consistent but judicious use of emails, SMS, and push notifications is the key to a highly engaged and loyal user base.
+
+Timely and personalized messages delivered to the users’ devices and inboxes help them stay informed and connected, even when they’re not actively using your app. This is where Messaging comes in, a product built to help you streamline email, SMS, and push notifications.
+
+## All your messages in one place
+
+As mentioned, Messaging covers every communication avenue under one unified API, allowing you to send email, SMS, and push notifications from your Appwrite project. It connects with a variety of third-party providers to deliver you messages. See this table to get an overview of what message types you can send with each provider.
+
+| Message Type      | Provider                                         |
+|-------------------|--------------------------------------------------|
+| Push Notifications| - APNs (Apple)                                   |
+|                   | - FCM (Google)                                   |
+| Email             | - Mailgun                                        |
+|                   | - Sendgrid                                       |
+|                   | - SMTP (Generic)                                 |
+| SMS               | - Twilio                                         |
+|                   | - MSG91                                          |
+|                   | - Telesign                                       |
+|                   | - TextMagic                                      |
+|                   | - Vonage                                         |
+
+
+## Message your existing users
+
+With Messaging, you can deliver messages to groups of users through topics or address individual users and their devices through targets. Messaging deeply integrates with integrated existing accounts in Appwrite out of the box. You can immediately start messaging your existing users and integrate deeply with other Appwrite features.
+
+You can address recipients of messages at three levels:
+
+- Topics, which hold a group of users’ devices.
+- Users which holds all targets associated with their account.
+- Targets, which are individual emails, phone numbers, and devices.
+
+You can publish announcements and newsletters through topics, deliver user-specific promotions and alerts to all users’ devices, and address individual targets for location-based or real-time alerts.
+
+![Creating a message from the Console](/images/blog/message-overview.png)
+
+## Deliver messages your way
+
+Draft, schedule, and send your messages in the Appwrite Console, or use Appwrite SDKs to send messages through code.
```

Comment:
I would love a bit more detail about the scheduling feature. Maybe mentioning some example use cases.